### ISSUE TITLE
TrellisDataset throws RIOT exception on instantiation

### ISSUE THREAD

**Issue by christopher-johnson** [trellis-ldp-archive/trellis-http#5]
This is an upstream issue.  https://issues.apache.org/jira/browse/COMMONSRDF-66

Reproduceable with this test, which is the same method used in JenaDatasetImpl:
```java
    @Test
    public void toStringTest() {
            DatasetGraph dg = DatasetGraphFactory.createGeneral();
            StringWriter sw = new StringWriter();
            RDFDataMgr.write(sw, dg, Lang.NT);
            String out = sw.toString();
        }
```

**Comment by acoburn** [comment]
@christopher-johnson do you have any information on where this error is being triggered? Is it the instantiation that is causing the error (as the title suggests here)? or is it the `toString` method that is causing it (as is suggested by the linked issue)? I am wondering if I could mitigate it with a custom `toString` class in the `TrellisDataset` class.

**Comment by christopher-johnson** [comment]
I made a [PR](https://github.com/apache/commons-rdf/pull/42).  AFAIK, you do not call `toString`, so it is just when `getInstance().createDataset()` creates the object that a "silent" RIOT exception is included.  Not sure how that can be avoided.

**Comment by acoburn** [comment]
OK, thanks for the update.

**Comment by acoburn** [comment]
Noting here that the upstream issue has been resolved and will be part of the 0.5.0 release of commons-rdf.
